pixart.js

```diff
@@ -1 +1,43 @@
+var brush = document.querySelector(".brush");
+var button = document.querySelector("#set-color");
+var colorInput = document.querySelector("#color-field");
+var color = "#1B4370";
```

Comment:
Maybe I missed this somewhere in the instructions or starter code, but is there a reason you assigned `"#1B4370"  `before grabbing the value of the user's input?